### PR TITLE
feat(formatter): Add prettypst config

### PR DESCRIPTION
**Pull request by GaetanLepage**
Add the [prettypst](https://github.com/antonWetzel/prettypst) formatter for Typst.

### CODE REVIEW

**Comment by GaetanLepage** [comment]
In the vscode instruction, they add `--file-location=${file}` to the command.
But for helix, they don't do it.

https://github.com/antonWetzel/prettypst#vscodium-or-vscode

**Comment by creativenull** [comment]
if it works without the `--file-location` then I don't see any problem with it.

File: doc/SUPPORTED_LIST.md
Comment (creativenull) [inline doc/SUPPORTED_LIST.md:1]: No need for this file, I will generate this in a separate commit

File: doc/supported-list.json
Comment (creativenull) [inline doc/supported-list.json:1]: Same for this file too, no need to include in the PR

**Review by creativenull — CHANGES_REQUESTED** [review]

File: doc/SUPPORTED_LIST.md
Comment (GaetanLepage) [reply doc/SUPPORTED_LIST.md:1]: Done !

File: doc/supported-list.json
Comment (GaetanLepage) [reply doc/supported-list.json:1]: Removed

**Review by creativenull — APPROVED** [review]

**Pull request merged**